docs/user-guide/docs-source/spawn-in-volume.md

```diff
@@ -1,10 +1,8 @@
 
-The **Spawn in Volume** actor works by using the position of the actor and the pre-made _box component_ attached to the actor to determine the rectangular area where obstacles will be spawned. If the bounded area contains multiple surface types, you can fine tune the **Spawn in Volume**'s behavior by specifying by <number-badge>2</number-badge> surface name pattern <number-badge>3</number-badge> and/or by tag (see [Ambit Spawner: How To Use Tags](/spawner-parent/#how-to-use-tags)). This will limit obstacles to spawn only on the surfaces that match your specifications within the bounded area. If you don't want to manually specify surfaces, there is also the option to <number-badge>1</number-badge> snap all spawned obstacles to _any_ surfaces below the bounded area. 
-
-**Spawn in Volume** will use the <number-badge>4</number-badge> _density_ range and the <number-badge>5b</number-badge> _rotation_ range you provide to instantiate copies of an Unreal Engine actor randomly selected from an <number-badge>6</number-badge> Array of Unreal Engine actors you've chosen within the box component's bounds. If you just want all spawned obstacles to face the same direction, you can select <number-badge>5a</number-badge> _Restrict to One Rotation_ and **Spawn in Volume** will automatically update the range's maximum to equal its minimum. To enable physics simulation on your spawned obstacles, just select <number-badge>#</number-badge> Add Physics. By default, the **Spawn in Volume** will remove overlaps, but this behavior can be turned off by unselecting <number-badge>#</number-badge> Remove Overlaps. To change the placement pattern, just adjust the <number-badge>7</number-badge> _random seed_ value.
+**Spawn in Volume** will use the <number-badge>3</number-badge> _density_ range and the <number-badge>5</number-badge> _rotation_ range you provide to instantiate copies of an Unreal Engine actor randomly selected from an <number-badge>7</number-badge> Array of Unreal Engine actors you've chosen within the box component's bounds. If you just want all spawned obstacles to face the same direction, you can select <number-badge>4</number-badge> _Restrict to One Rotation_ and **Spawn in Volume** will automatically update the range's maximum to equal its minimum. To enable physics simulation on your spawned obstacles, just select <number-badge>6</number-badge> Add Physics. By default, the **Spawn in Volume** will remove overlaps, but this behavior can be turned off by unselecting <number-badge>8</number-badge> Remove Overlaps. To change the placement pattern, just adjust the <number-badge>9</number-badge> _random seed_ value.
```

Comment:
"...Array of Unreal Engine actors...". "Array" is capitalized but "actors" isn't. We should be consistent. I suggest user lower case in most cases unless referring to a specific subclass type. So change "Array" -> "array" here.

Reply:
In most places, when referring to a param by it's label test we've been italicizing that text. Ex. "_Restrict to One Rotation_". But "Add Physics" hasn't been italicized here. Please italicize it both for consistency and better readability. Same applies to "Remove Overlaps".

Reply:
Capitalize "random seed" (number 9) to be consistent with other callouts.